packages/common/http/src/transfer_cache.ts

```diff
@@ -91,11 +94,14 @@ export function transferCacheInterceptorFn(
   req: HttpRequest<unknown>,
   next: HttpHandlerFn,
 ): Observable<HttpEvent<unknown>> {
+  const injector = inject(EnvironmentInjector);
   const {isCacheActive, ...globalOptions} = inject(CACHE_OPTIONS);
   const {transferCache: requestOptions, method: requestMethod} = req;
+  const isBrowser = isPlatformBrowser(injector.get(PLATFORM_ID));
```

Comment:
Interceptors are invoked in the injection context, so we can use the `inject()` method directly, without injecting an injector first, i.e.:
```suggestion
  const {isCacheActive, ...globalOptions} = inject(CACHE_OPTIONS);
  const {transferCache: requestOptions, method: requestMethod} = req;
  const isBrowser = isPlatformBrowser(inject(PLATFORM_ID));
```